lib/rubocop/cop/style/option_hash.rb

```diff
@@ -50,6 +53,11 @@ def on_args(node)
         def supports_keyword_arguments?
           RUBY_VERSION >= '2.0.0'
         end
+
+        def name_in_suspicious_param_names?(arg_name)
+          cop_config.key?('SuspiciousParamNames') &&
+            cop_config['SuspiciousParamNames'].include?(arg_name.to_s)
```

Comment:
I'm pretty sure it merges, otherwise enabling/disabling wouldn't really work, since those are just a config option too.